lib/taurus/qt/qtgui/panel/taurusvalue.py

```diff
@@ -442,6 +445,14 @@ def onSetFormatter(self):
         if hasattr(self._readWidget, 'onSetFormatter'):
             return self._readWidget.onSetFormatter()
 
+    def setFormat(self, format):
```

Comment:
I think that format is a Python built-in. In this case I would suggest to use `format_`